test/pages/pages.test.js

```diff
@@ -118,9 +118,9 @@ async function getMostVisited() {
 async function getTestSetup() {
   const mostVisitedUrls = await getMostVisited();
 
-  mostVisitedUrls.push({
-    req_url: 'https://theblog--adobe.hlx.page/en/publish/2019/11/22/asia-pacific-insiders-share-their-inspiration-from-max-2019.html',
-  });
+  // mostVisitedUrls.push({
+  //   req_url: 'https://theblog--adobe.hlx.page/en/publish/2019/11/22/asia-pacific-insiders-share-their-inspiration-from-max-2019.html',
+  // });
```

Comment:
why not just disable the diff tests in the `circleci/config.yaml` ?

Reply:
I wanted to see if this is still green (nothing else break since yesterday and other tests still run). Like this, I do not cut everything. 
My new plan is anyway to revert those changes (including the hash exclusion) right after the release. Like this we can track new coming hash issues.